lib/beaker/hypervisor/container_podman.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'beaker/hypervisor/docker'
+
+module Beaker
+  class ContainerPodman < Beaker::Docker
```

Comment:
mhm or should it inherit from Beaker::Container?

Reply:
Technically, `Beaker::Container`, not that it makes a difference today…